src/ColorContextMenu-Core.package/RadialMenuItemMorph.class/instance/initialize.st

```diff
@@ -5,6 +5,7 @@ initialize
 	
 	self privateInteractionState: nil.
 	self borderWidth: self defaultBorderWidth.
+	self addDependent: self.
 	
 	self addMorph: (labelMorph := self basicLabelMorph).
-	labelMorph color: Color white.
\ No newline at end of file
+	"labelMorph color: Color white."
```

Comment:
Is this WIP or can you remove this line?

Reply:
removed